src/components/common/AudioLibrarySearchField.vue

```diff
@@ -69,102 +70,102 @@
 </template>
 
 <script>
-	import fuse from 'fuse.js'
```

Comment:
@poojahpatil90 , thanks for thinking of these indentation etc, but I would avoid making auto fix like this before PR review. This is a big distraction for reviewer especially for big PR.

My suggestion is to wait for PR approval, then commit on last change to make auto fix from IDE before merge, what do you think?

Reply:
@safydy-r  Sorry about the indentation fixes. My IDE was setup to auto fix the ESLint issues, which I did not realize until a later point and it was difficult to revert back those changes. 
One solution for now is to hide white-space changes from the settings button above(as suggested by Brett)